Extensions/tweaks.js

```diff
@@ -570,39 +581,6 @@ XKit.extensions.tweaks = new Object({
 			"xkit_tweaks_hide_follower_count");
 		}
 
-		if (XKit.extensions.tweaks.preferences.wrap_tags.value &&
```

Comment:
We should probably keep this code around for at least a little bit in case there are tumblr pages that don't have the new behavior yet. (`/submit`, maybe?)

Reply:
nevermind